fastplotlib/graphics/line_collection.py

```diff
@@ -63,6 +65,13 @@ def __init__(
 
         kwargs
             passed to GraphicCollection
+
+        Examples
+        --------
+        .. code-block::python
+
```

Comment:
simple examples that also demonstrate how graphic features are indexable, can just copy from `simple.ipynb` with descriptions